package.json

```diff
@@ -40,8 +40,11 @@
   "dependencies": {
     "@babel/polyfill": "^7.7.0",
     "bignumber.js": "^8.1.1",
+    "bip32": "^2.0.5",
```

Comment:
This was being used but via another dependency. I wanted to be explicit about our usage to be sure of the version and API we were using.